Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -613,6 +613,31 @@ public UserDto GetUserDto(User user, string remoteEndPoint = null)
             return dto;
         }
 
+        public PublicUserDto GetPublicUserDto(User user, string remoteEndPoint = null)
+        {
+            if (user == null)
+            {
+                throw new ArgumentNullException(nameof(user));
+            }
+
+            bool hasConfiguredPassword = GetAuthenticationProvider(user).HasPassword(user);
+            bool hasConfiguredEasyPassword = !string.IsNullOrEmpty(GetAuthenticationProvider(user).GetEasyPasswordHash(user));
+
+            bool hasPassword = user.Configuration.EnableLocalPassword &&
+                !string.IsNullOrEmpty(remoteEndPoint) &&
+                _networkManager.IsInLocalNetwork(remoteEndPoint) ? hasConfiguredEasyPassword : hasConfiguredPassword;
```

Comment:
```suggestion
                _networkManager.IsInLocalNetwork(remoteEndPoint) ? (hasConfiguredEasyPassword || hasConfiguredPassword) : hasConfiguredPassword;
```
As I think that if I haven't configured an easy password, I should still be using a normal password even in LAN, no?

Reply:
Honestly you're right, but I followed the logic of the other methods, e.g. `GetUserDto`. Should I change that one too? Otherwise we risk to have inconsistencies between the result of the APIs that return an `User` and the ones that return `PublicUser` instead.

Reply:
@jellyfin/backend @jellyfin/web anyone has any input on that?

Reply:
Would it be feasible to make it configurable? I can see both sides of requiring pass auth on the Lan. Otherwise, yeah, it might for the sake of not having  subtle bugs, to always require the password.

Reply:
Disregard, it's me not knowing how to read code probably... It is exactly what `user.Configuration.EnableLocalPassword` does - allows me to have empty password when on LAN and not configured an "Easy PIN".